hazelcast/src/test/java/com/hazelcast/client/topic/ClientReliableTopicOnClusterRestartTest.java

```diff
@@ -154,35 +155,36 @@ public void shouldContinue_OnClusterRestart_whenDataLoss_LossTolerant_afterInvoc
     }
 
     @Test
-    public void shouldFail_OnClusterRestart_whenDataLoss_notLossTolerant() throws InterruptedException {
-        HazelcastInstance member = hazelcastFactory.newHazelcastInstance(smallInstanceConfig());
+    public void shouldFail_OnClusterRestart_whenDataLoss_notLossTolerant() {
+        Config config = smallInstanceConfig();
+        String topicName = "topic";
+        config.getRingbufferConfig(topicName)
+              .setCapacity(10);
+
+        HazelcastInstance member = hazelcastFactory.newHazelcastInstance(config);
         ClientConfig clientConfig = new ClientConfig();
         clientConfig.getConnectionStrategyConfig().getConnectionRetryConfig().setClusterConnectTimeoutMillis(Long.MAX_VALUE);
         int invocationTimeoutSeconds = 2;
         clientConfig.setProperty(ClientProperty.INVOCATION_TIMEOUT_SECONDS.getName(), String.valueOf(invocationTimeoutSeconds));
```

Comment:
Can we get rid of `INVOCATION_TIMEOUT_SECONDS` also? It does not seem to have any value anymore.

Reply:
Sure, didn't notice it.